5_analyze_data/notebooks/KS-test_analysis/nf1_ks_test.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Perform Two-Sample KS-Test on NF1 Genotype Samples
+
+# ## Import libraries
+
+# In[1]:
+
+
+import numpy as np
+from scipy.stats import ks_2samp
+import pathlib
+import pandas as pd
+
+
+# ## Set seed
+
+# In[2]:
+
+
+np.random.seed(0)
+
+
+# ## Load in NF1 data
+
+# In[3]:
+
+
+norm_fs_data = pathlib.Path("../../../4_processing_features/data/nf1_sc_norm_cellprofiler.csv.gz")
+
+data = pd.read_csv(norm_fs_data, compression="gzip", index_col=0)
+
```

Comment:
it is good practice to add the following lines after reading in data:

```python
print(data.shape)
data.head()
```

Doing this helps a reader easily understand the data you're working with

Reply:
:+1: Added now and I will get into this habit.